packages/audit/scripts/log-viewer.py

```
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+"""Audit log viewer: 統一イベントログをフィルタ・表示する。
+
+Usage:
+  python log-viewer.py                          # 全イベント表示
+  python log-viewer.py --type cli_call          # タイプ別フィルタ
+  python log-viewer.py --session SID            # セッション別フィルタ
+  python log-viewer.py --limit 50               # 件数制限（デフォルト: 最新 100）
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)